rpc/types/src/misc/tx_entry.rs

```diff
@@ -0,0 +1,59 @@
+//! TODO
+
+//---------------------------------------------------------------------------------------------------- Use
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+#[cfg(feature = "epee")]
+use cuprate_epee_encoding::{
+    epee_object,
+    macros::bytes::{Buf, BufMut},
+    EpeeValue, Marker,
+};
+
+//---------------------------------------------------------------------------------------------------- TxEntry
+#[doc = crate::macros::monero_definition_link!(
+    cc73fe71162d564ffda8e549b79a350bca53c454,
+    "rpc/core_rpc_server_commands_defs.h",
+    389..=428
+)]
+/// Used in [`crate::other::GetTransactionsResponse`].
+#[derive(Clone, Debug, Default, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct TxEntry {
+    pub as_hex: String,
+    pub as_json: String,
+    pub block_height: u64,
+    pub block_timestamp: u64,
+    pub confirmations: u64,
+    pub double_spend_seen: bool,
+    pub in_pool: bool,
+    pub output_indices: Vec<u64>,
+    pub prunable_as_hex: String,
+    pub prunable_hash: String,
+    pub pruned_as_hex: String,
+    pub received_timestamp: u64,
+    pub relayed: bool,
+    pub tx_hash: String,
+}
+
+// TODO: custom epee
+// <https://github.com/monero-project/monero/blob/cc73fe71162d564ffda8e549b79a350bca53c454/src/cryptonote_protocol/cryptonote_protocol_defs.h#L406-427>
```

Comment:
Is this the correct link?

Reply:
Oops, fixed to correct link (https://github.com/monero-project/monero/blob/cc73fe71162d564ffda8e549b79a350bca53c454/src/rpc/core_rpc_server_commands_defs.h#L406-L428).